EIPS/eip-xtndr-family.md

```diff
@@ -0,0 +1,98 @@
+
+An example of a variadic arithmetic sequence would be:
+
+XTNDR 11 ADD 4 17 8 10 66 9 2 2 7 1 3 2
+
+which performs addition over 11 values (in the usual order as if pairwise evaluated in sequential order), equivalent to 4 + 17 + 8 + 10 + 66 + 9 + 2 + 2 + 7 + 1 + 3 + 2. This returns a value of 131. 
+
+This may be applied to a logical operation over more operands than currently supported. An example of the assembly for extending the greater-than operation (0x11) to a sequence of 6 numbers to check if it is a monotonically decreasing sequence is:
+
+XTNDR 6 GT A B C D E F
+
+This could be considered a compact equivalent to:
+
+GT A B AND
+GT B C AND 
+GT C D AND
+GT D E AND
+GT E F
+
+Any 'extended sequence' cannot contain JUMPs or stack manipulation opcodes, with the exception of potential usages to implement EIP-663.
+
+## Rationale
+
+Opcode address rational:
+The E-series of opcodes is currently empty aside from 0xEF, which may be interpreted as standing for 'Ethereum (Object) Format'. The EE opcode is preferably reserved for future usages pertaining to 'Execution Environment'. 
+The choice of EA could be interpreted as 'Extended Address'.
+The 'macro memory' is preferred to be a contiguous portion of address (opcode) space, however this is not itself necessary.
+
+Greatly increasing the amount of opcode space available will substantially reduce the consequences of adding niche, application-specific opcodes or those which are of speculative value. This may eventually allow replacing precompiles with multi-byte opcodes if this proves more efficient, or allowing parts of already implemented precompiles to be represented as multi-byte opcodes to allow for more modular cryptographic techniques.
+
+This method is in some respects simpler than other proposals for improving the 'bytecode density' of the EVM ISA, whilst also being broadly compatible with those other proposals. 
+
+Additionally, as gas accounting imposes a non-negligible overhead during execution, storing the gas costs of particular predictable sequences in their own (exclusive) area of memory may be more efficient.
+
+## Backwards Compatibility
+
+## Test Cases
+
+## Reference Implementation
+
+## Security Considerations
+
+Due to the significant changes made to execution, the likelihood of mispricing small sequences of bytecode which require heavy operations ("EVM bombs") is high if not approached carefully. There are several methods of implementation, some of which may be considerably safer (in terms of memory requires and principally time of execution). 
+
+These changes may have intense interactions with EIP-615 and EIP-2315, among others.
```

Comment:
```suggestion
```
EIP-615 is stagnant and EIP-2315 is still a draft, so I recommend not mentioning them here as that would create a dependency on those EIPs which I don't think you want.